plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -166,26 +165,6 @@ export class EventPipelineRunner {
         }
     }
 
-    async runAppsOnEventPipeline(event: PostIngestionEvent): Promise<EventPipelineResult> {
-        try {
-            this.hub.statsd?.increment('kafka_queue.event_pipeline.start', { pipeline: 'onEvent' })
-            await this.runStep(processOnEventStep, [this, event], event.teamId, false)
-            this.hub.statsd?.increment('kafka_queue.onevent.processed')
-            return this.registerLastStep('processOnEventStep', event.teamId, [event])
-        } catch (error) {
-            if (error instanceof DependencyUnavailableError) {
```

Comment:
Do we need to keep that error handling around, to avoid breaking the consumer on deploy? Is the value of `silentFailuresAsyncHandlers` strictly zero?

Reply:
yup http://grafana-prod-us/explore?panes=%7B%22wD6%22:%7B%22datasource%22:%22victoriametrics%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22count%28async_handlers_silent_failure%7Bcontainer%3D%5C%22ingestion-plugins-onevent%5C%22%7D%29%22,%22range%22:true,%22datasource%22:%7B%22type%22:%22prometheus%22,%22uid%22:%22victoriametrics%22%7D,%22editorMode%22:%22code%22,%22useBackend%22:false,%22disableTextWrap%22:false,%22fullMetaSearch%22:false,%22includeNullMetadata%22:true%7D%5D,%22range%22:%7B%22from%22:%22now-7d%22,%22to%22:%22now%22%7D%7D%7D&schemaVersion=1&orgId=1 
also we catch the errors https://github.com/PostHog/posthog/blob/731504fa0a3a05f8d0c061975b6ecb62399da653/plugin-server/src/worker/plugins/run.ts#L35 from any onEvent within plugin-server and errors before that probably would be bugs in our code 🤷‍♀️  so good to take kafka lag on those